site/en/blog/new-in-chrome-104/index.md

```diff
@@ -0,0 +1,210 @@
+
+Check out Rachel’s article
+[New syntax for range media queries in Chrome 104](/blog/media-query-range-syntax/)
+for more details.
+
+## Shared Element Transitions start new origin trial {: #shared-element-transitions }
+
+Platform specific apps typically have smooth transitions between different views,
+they look beautiful, they keep the user in context, and they help the
+experience feel more performant. Whereas on the web, a full navigation can be
+harsh, and sometimes means a momentary blank screen. For a single page app,
+it can be better, but transitions are still hard.
+
+Shared Element Transitions, starting a new [origin trial][set-ot] in Chrome
+104, allow you to provide smooth transitions, regardless of whether the
+transitions are cross-document (for example in a multi-page app), or
+intra-document (for example in a single page app).
+
+Here’s a rough example of how transitions work for a single page app. In the
+navigate function, get the new page content, then check to see if transitions
+are supported, if not, update the page without a transition. If they are,
+create a `transition()` and call `start()` on it, letting the API know when
+the DOM change is complete.
+
+```js
+async function spaNavigate(path) {
+  // Get new page content.
+  const data = await fetchPage(path);
+
+  // Check if transitions are supported, if not, use classic method.
+  if (!document.createDocumentTransition) {
+    await updateDOM(data);
+    return;
+  }
+
+  // Create transition
+  const transition = document.createDocumentTransition();
+
+  // Start transition, let API know when DOM change is complete.
+  transition.start(() => updateDOM(data));
+}
+```
+
+{% Columns %}
+{% Column %}
+Under the hood, Shared Element Transitions use CSS Animations, so you can
+change from a fade in effect, to slide in, or whatever you want.
+{% endColumn %}
+
+{% Column %}
+{% Video src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/6qxo2mCaOZH887Lw75VU.mp4", autoplay="true", loop="true", muted="true" %}
+{% endColumn %}
+{% endColumns %}
+
+I’ve just scratched the surface, so check out Jake’s video
+[Bringing Page Transitions to the Web](https://www.youtube.com/watch?v=JCJUPJ_zDQ4),
+or dive into the [explainer][set-explainer].
+
+## And more! {: #more }
+
+Of course there's plenty more.
+
+* When cookies are set with an explicit `Expires` or `Max-Age` attribute,
+  the value will now be capped to no more than 400 days.
+* There are enhancements to the multi-screen window placement API.
+* And the `overflow-clip-margin` CSS property specifies how far an element's
+  content is allowed to paint before being clipped.
+
+## Further reading
+
+This covers only some of the key highlights. Check the links below for
+additional changes in Chrome 104.
+
+* [What's new in Chrome DevTools (104)](/blog/new-in-devtools-104/)
+* [Chrome 104 deprecations and removals](/blog/deps-rems-104/)
+* [ChromeStatus.com updates for Chrome 104](https://www.chromestatus.com/features#milestone%3D104)
```

Comment:
I'm going back to make sure everything in 104 was marked on Chrome Status. This doesn't always happen (which is why I formed a committee).